docs/src/index.md

```diff
@@ -0,0 +1,457 @@
+# Introduction
+
+Metaprogramming tools for DataFrames.jl objects.
+These macros improve performance and provide more convenient syntax.
+
+DataFrames.jl has the functions `select`, `transform`, and `combine` 
+for manipulating data frames. DataFramesMeta provides the macros 
+`@select`, `@transform`, and `@combine` to mirror these functions with 
+more convenient syntax. Inspired by [dplyr](https://dplyr.tidyverse.org/) in R 
+and [LINQ](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/linq/)
+in C#. 
+
+In addition, DataFramesMeta provides 
```

Comment:
```suggestion
In addition, DataFramesMeta.jl provides 
```

Reply:
same in other places

Reply:
Trying to only use the `.jl` in introductions, and just the package name after.

Reply:
OK - we can go this way if you prefer